src/RPVoiceChatServer.cs

```diff
@@ -39,6 +39,7 @@ public override void StartServerSide(ICoreServerAPI api)
             WorldConfig.Set(VoiceLevel.Shouting, WorldConfig.GetInt(VoiceLevel.Shouting));
             WorldConfig.Set("force-render-name-tags", WorldConfig.GetBool("force-render-name-tags", true));
             WorldConfig.Set("encode-audio", WorldConfig.GetBool("encode-audio", true));
+            WorldConfig.Set("wall-thickness-weighting", WorldConfig.GetFloat("wall-thickness-weighting", 1.5f));
```

Comment:
Previous default value was 2. I don't mind reducing the strength of muffling but it's quite a noticeable change and is hard to justify
```suggestion
            WorldConfig.Set("wall-thickness-weighting", WorldConfig.GetFloat("wall-thickness-weighting", 2));
```

Reply:
After testing, "2" had no significant effect with one layer of blocks.
The idea was to have a default value that affect significantly sound with only one layer because most players will have only "one layer" walls.
I can undo it, if you want.

Reply:
I believe it was decided to not muffle the sound too much because on RP servers when people have conversation and, for example, a store owner goes in a back room to fetch something it would be too inaudible and overall inconvenient to have. Some decorations and non-full blocks will also cause muffling, so it would be weird if a cup sitting on the table had significant impact on player's voice
Now that it is adjustable people can increase/lower the muffling strength if they want, so I suggest we keep the default at 2 as it already was

Reply:
Ok ! done 👍